sources/dynamodb/table.go

```diff
@@ -199,13 +199,9 @@ func NewTableSource(client Client, accountID string, region string) *sources.Alw
 				return nil, errors.New("cannot map nil output")
 			}
 
-			inputs := make([]*dynamodb.DescribeTableInput, len(output.TableNames))
+			inputs := make([]*dynamodb.DescribeTableInput, 0, len(output.TableNames))
 
 			for i := range output.TableNames {
-				if output.TableNames[i] == "" {
-					continue
-				}
-
```

Comment:
Sidenote: the append pattern also means it would be safe to skip entries here as append will continue to fill up the array densely, while otherwise skipping and assigning to `output[i]` would leave gaps in the array.

Reply:
Ooh that's a good point, we might end up allocating slightly more memory than we need but that's unlikely to be a big deal